src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -30,69 +31,26 @@ export class SearchMenu extends React.Component {
     const dropdownInputField = document.getElementById(
       'search-header-dropdown-input-field',
     );
+
     if (isOpen && !prevProps.isOpen && dropdownInputField) {
       dropdownInputField.focus();
     }
   }
 
-  handleInputChange = event => {
```

Comment:
`handleInputChange` and `handleSearchEvent` aren't used; I checked the rest of this file and they aren't referenced.

Regression tested searching through the header because I had to check the analytics there too.